README.rst

```diff
@@ -73,6 +73,19 @@ If you wish to set a threshold for iviewer that is *lower* than for the server:
 NB: Z-projection is not supported for tiled images in OMERO
 (Images larger than 2048 * 2048 pixels per plane are tiled in iviewer).
 
+OMERO uses Spectrum Color Picker for selecting ROI colors. 
+The roi_color_palette option allows you to specify a grid of colors for users to choose for ROIs.
+Define rows with brackets, and use commas to seperate values. 
+You can only view the first item of each row without enabling show_palette_only.
+To define a color palette use::
+    
+    $ omero config set omero.web.iviewer.roi_color_palette "[rgb(0,255,0)],[darkred,red,pink],[#0000FF]"
+  
+Hides the default color picker.
```

Comment:
"Hides the default color picker." This seems strange when you're referring to the setting below (this isn't a title).
Maybe: "To hide the default color picker (and show a grid for the color palette), set show_palette_only to true".